src/utils/errors.js

```diff
@@ -14,30 +14,26 @@
  * limitations under the License.
  */
 
-const {
-  createAbortError,
-  isAbortError,
-} = require('web-activities/activity-ports');
-
 /**
  * Whether the specified error is an AbortError type.
  * See https://heycam.github.io/webidl/#aborterror.
  * @param {*} error
  * @return {boolean}
  */
 export function isCancelError(error) {
-  return isAbortError(error);
+  return error?.name === 'AbortError';
 }
 
 /**
- * Creates or emulates a DOMException of AbortError type.
+ * Creates an Error of AbortError type.
  * See https://heycam.github.io/webidl/#aborterror.
- * @param {!Window} win
  * @param {string=} message
```

Comment:
nit (existing issue): the type should just be `string` since there is no default value.

Reply:
Nice catch, updated